Libraries/LibCompress/BrotliDictionary.cpp

```diff
@@ -8,25 +8,36 @@
 #include <LibCompress/BrotliDictionary.h>
 
 // Include the 119.9 KiB of dictionary data from a binary file
+
+// At time of writing GCC does not support disabling the warning so would not work with pedantic on
```

Comment:
```suggestion
// GCC v.X.Y.Z does not support disabling the warning so would not work with pedantic on
```
would be better – i might forget when the time of writing exactly was.